### ISSUE TITLE
When the device dpi is equal to 1, the font is blurred, but it displays normally on the retina screen

### ISSUE THREAD

**Issue by Malenconiaprincep** [ONLYOFFICE/DocumentServer#1313]
**Do you want to request a *feature* or report a *bug*?**
bug
**What is the current behavior?**

When the device dpi is equal to 1, the font is blurred, but it displays normally on the retina screen

eg:  dpi 1
![image](https://user-images.githubusercontent.com/6262739/123731397-9f261a80-d8ca-11eb-83b8-e65fe8ba800f.png)

eg: dpi 2
![image](https://user-images.githubusercontent.com/6262739/123731458-bb29bc00-d8ca-11eb-85bf-057bb88736b5.png)

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

**What is the expected behavior?**

They should be same

**Did this work in previous versions of DocumentServer?**

**DocumentServer version:**
6.3.0
**Operating System:**
mac os 11.2
**Browser version:**
chrome 91.0.4472.114

**Comment by ShockwaveNN** [comment]
Hi, do you have custom zoon in browser?
That is your device you've testing it?

**Comment by Malenconiaprincep** [comment]
Browser does not zoom

**Comment by ShockwaveNN** [comment]
My second question?

That is your device you've testing it?
What's its resolution and display size?

**Comment by Malenconiaprincep** [comment]
Yes, Use the monitor to split the Mac， display size is  1920 * 1200

**Comment by DenisDeeSign** [comment]
Hello @Malenconiaprincep,
could you attach a screenshoot with second monitor scaling? You can find it at System Preferences > Displays > Display tab.

**Comment by Malenconiaprincep** [comment]
![image](https://user-images.githubusercontent.com/6262739/124210120-7a75b100-db1d-11eb-811d-60c3e5a11803.png)

**Comment by DenisDeeSign** [comment]
What Font Hinting in Advanced Settings do you use?
